src/errorReporting.js

```diff
@@ -96,10 +96,19 @@ function getPendingPromise(state) {
   return state.getIn(['errorReporting', 'pendingPromise']);
 }
 
-export function serverSideError(error) {
-  // nodejs console truncates output by default
-  // https://nodejs.org/api/util.html#util_util_inspect_object_options
-  console.error(util.inspect(error, false, 10, true));
+export function serverSideError(queue) {
+  [].concat(queue).forEach((raw) => {
+    const {
+      msg, stack, type, otherData,
+    } = raw;
+    const err = new Error(msg);
+    Object.assign(err, {
+      name: type,
```

Comment:
I suspect overriding the name property of the Error instance will cause some surprises during debugging.

Reply:
not doing that anymore, however would've been nice to have an error name for the server error similar to `ClientReportedError`, currently server errors will show the name property as`Error`